contracts/forwarder/Eip712Forwarder.sol

```diff
@@ -12,23 +12,17 @@ contract Eip712Forwarder is IForwarder {
 
     //all valid requests must start with this prefix.
     // request name is arbitrary, but the parameter block must match exactly these parameters
-    string public constant GENERIC_PARAMS = "_ForwardRequest request";
-    string public constant GENERIC_TYPE = "_ForwardRequest(address to,bytes data,address from,uint256 nonce,uint256 gas)";
-    bytes32 public constant GENERIC_TYPEHASH = keccak256(bytes(GENERIC_TYPE));
+    string public constant GENERIC_PARAMS = "address to,bytes data,address from,uint256 nonce,uint256 gas";
```

Comment:
I swear I have no idea why, but these parameters come in a different order in my head 😂
For me it is 'from, to, gas, value, nonce, data'? I know there is no standard or anything...
Also, if you see it in RLP hex having variable size parameter last is more... clear. Or maybe I am just slowly going crazy.